pipeline/process/DatabaseBuilder.ts

```diff
@@ -315,19 +313,15 @@ export class DatabaseBuilder {
                         mentionsCount[mentionIdx] = (mentionsCount[mentionIdx] || 0) + 1;
                     } else if (tag === "url") {
                         // TODO: transform URL only messages to attachments
-                        try {
-                            const hostname = new URL(text).hostname.toLowerCase();
-                            let domainIdx = this.domains.getIndex(hostname);
-                            if (domainIdx === undefined) domainIdx = this.domains.set(hostname, hostname);
-                            domainsCount[domainIdx] = (domainsCount[domainIdx] || 0) + 1;
-                        } catch (ex) {}
```

Comment:
This try must be preserved. `new URL(text)` might fail.

Reply:
Oh, I was thinking it was incomplete because of the empty braces, I've never seen an empty catch block in Javascript so I was thinking how in Python you need to do 'pass' to just ignore it